### PR TITLE
[MFTF] Reduce performance overhead on Acceptance tests

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
Another huge performance optimization, this time I focus on Acceptance tests

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)
N/A

### Questions or comments
- Reduce useless `<wait>` time
- Replace `cache:flush` with `cache:clean <type>`

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Sales/Test/Mftf/Test/AdminAvailabilityCreditMemoWithNoPaymentTest.xml [hunk 1]
```diff
@@ -25,19 +25,19 @@
             </createData>
             <!-- Enable *Free Shipping* -->
             <createData entity="FreeShippingMethodsSettingConfig" stepKey="freeShippingMethodsSettingConfig"/>
-            <magentoCLI command="cache:flush" stepKey="flushCache"/>
+            <magentoCLI command="cache:clean config" stepKey="flushCache"/>
```
Comment (nuzil) [inline app/code/Magento/Sales/Test/Mftf/Test/AdminAvailabilityCreditMemoWithNoPaymentTest.xml]: Is it possible also rename stekKey to more meanfull?

Comment (lbajsarowicz) [reply app/code/Magento/Sales/Test/Mftf/Test/AdminAvailabilityCreditMemoWithNoPaymentTest.xml]: I'd love to do it. Sure, will commit with other improvements.

**Comment by lbajsarowicz** [comment]
@magento run Functional Tests B2B, Unit Tests

**Comment by lbajsarowicz** [comment]
Ready to proceed with CR and merge to `2.4-develop`

**Comment by lenaorobei** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Comment by lbajsarowicz** [comment]
@magento run all tests

**Pull request closed without merging**